tchannel/tornado/peer.py

```diff
@@ -102,7 +103,7 @@ def __init__(self, tchannel, hostport, rank=None, on_conn_change=None):
         #: Collection of all connections for this Peer. Incoming connections
         #: are added to the left side of the deque and outgoing connections to
         #: the right side.
-        self._connections = deque()
+        self.connections = deque()
```

Comment:
Because the `@property` version of this was `list(self._connections)` which can be a little expensive. The callers don't mutate or attempt to mutate this. In fact, the only place this is accessed externally is in PeerStrategy, which uses it to check if the peer even has any connections.